includes/Hooks.php

```diff
@@ -132,18 +132,20 @@ public static function onBeforeCreateEchoEvent(
 		&$notifications, &$notificationCategories, &$icons
 	) {
 		$notificationCategories['wiki-creation'] = [
-			'priority' => 3,
+			'priority' => 1,
 			'tooltip' => 'echo-pref-tooltip-wiki-creation',
+			'no-dismiss' => [ 'all' ]
 		];
 
 		$notificationCategories['request-declined'] = [
-			'priority' => 3,
+			'priority' => 2,
 			'tooltip' => 'echo-pref-tooltip-wiki-request-declined'
 		];
 
 		$notificationCategories['request-moredetails'] = [
-			'priority' => 3,
-			'tooltip' => 'echo-pref-tooltip-wiki-request-moredetails'
+			'priority' => 1,
```

Comment:
This and all others can stay priority 3 or 2 if you really want it doesn't really have any affect here anyway.